sonarts-sq-plugin/sonar-typescript-plugin/sonarts-core/package-lock.json

```diff
@@ -6,19 +6,19 @@
   "dependencies": {
     "ansi-regex": {
       "version": "2.1.1",
-      "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-2.1.1.tgz",
+      "resolved": "https://repox.sonarsource.com/api/npm/npm/ansi-regex/-/ansi-regex-2.1.1.tgz",
```

Comment:
@inverno ?!

Reply:
@vilchik-elena holy mistery. Will investigate towards the end